backend/sportsmatch/src/main/java/com/sportsmatch/models/Token.java

```diff
@@ -0,0 +1,26 @@
+package com.sportsmatch.models;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Setter
+@Getter
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Token {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String token;
+
+    @Enumerated(EnumType.STRING)
+    private TokenType tokenType;
+    private boolean isExpired;
+    private boolean isRevoked;
```

Comment:
Is there a reason to differentiate between revoked and expired? Could we just have one field that says isValid?